subprojects/kotlin-dsl-plugins/src/main/kotlin/org/gradle/kotlin/dsl/plugins/base/KotlinDslBasePlugin.kt

```diff
@@ -45,8 +45,7 @@ abstract class KotlinDslBasePlugin : Plugin<Project> {
     override fun apply(project: Project): Unit = project.run {
         apply<EmbeddedKotlinPlugin>()
 
-        val kotlinDslPluginOptions = extensions.create("kotlinDslPluginOptions", KotlinDslPluginOptions::class.java)
-        kotlinDslPluginOptions.jvmTarget.convention("1.8")
+        extensions.create("kotlinDslPluginOptions", KotlinDslPluginOptions::class.java)
```

Comment:
This has the potential of being breaking, since setting the value to `null` will no longer restore the convention. Not that I like that behavior.
Which means the upgrade guide entry should mention this as well as the deprecation.

Reply:
Good point. I clarified the case with a change in behavior in the upgrading guide.